packages/web-app-files/src/composables/actions/files/useFileActionsEmptyTrashBin.ts

```diff
@@ -89,7 +89,7 @@ export const useFileActionsEmptyTrashBin = ({ store }: { store?: Store<any> } =
           return false
         }
 
-        return true
+        return store.getters['Files/activeFiles'].length > 0
```

Comment:
This will hide the button, we prefer to just disable it. You should be able to use the `isDisabled` property for this instead of `isEnabled` (a bit confusing, I know 😅 )

Reply:
Got it :smile:. Fixed